gradle.properties

```diff
@@ -2,7 +2,7 @@ projectVersion=3.2.0-SNAPSHOT
 projectGroup=io.micronaut.views
 micronautVersion=3.3.4
 micronautTestVersion=3.0.5
-groovyVersion=3.0.10
+groovyVersion=3.0.9
```

Comment:
Should be 3.0.10

Reply:
Resolved via https://github.com/micronaut-projects/micronaut-views/pull/365/commits/5e209423a733b6e8413f053947f54a7d1db2d72e